ngraph/test/type_prop/batch_norm.cpp

```diff
@@ -69,7 +70,8 @@ TEST(type_prop, batch_norm_inference_partial_input_rank_static_dynamic_ok)
     auto mean = make_shared<op::Parameter>(mean_et, mean_shape);
     auto variance = make_shared<op::Parameter>(variance_et, variance_shape);
 
-    auto bn = make_shared<op::BatchNormInference>(data_batch, gamma, beta, mean, variance, epsilon);
+    auto bn =
+        make_shared<op::v5::BatchNormInference>(data_batch, gamma, beta, mean, variance, epsilon);
```

Comment:
Why we remove tests for v0 operation?

Reply:
Agreed, will add new tests for v5 BatchNormInference.

Reply:
Fixed